src/Directory.Build.props

```diff
@@ -47,8 +47,10 @@
 
   <PropertyGroup Condition="'$(Language)' == 'C#'">
     <CheckForOverflowUnderflow>false</CheckForOverflowUnderflow>
-    <LangVersion>7.3</LangVersion>
+    <LangVersion>8.0</LangVersion>
     <Features>strict</Features>
+    <NullableReferenceTypes>true</NullableReferenceTypes>
+    <WarningsNotAsErrors>CS8625</WarningsNotAsErrors>
```

Comment:
>CS8625 [](start = 25, length = 6)

Why tune down this diagnostic?
The diagnostic that we're expecting people to tune down or disable is CS8600.

Reply:
I think this might be another issue.  Our CI normally builds with `/warnaserror` but this seems to be the only warning that gets promoted?

Reply:
N/M - the others are too.